dbms/src/Flash/executeQuery.h

```diff
@@ -16,9 +16,11 @@
 
 #include <Core/QueryProcessingStage.h>
 #include <DataStreams/BlockIO.h>
+#include <Flash/Executor/QueryExecutor.h>
 #include <Interpreters/Context.h>
 
 namespace DB
 {
 BlockIO executeQuery(Context & context, bool internal = false);
```

Comment:
Can we delete executeQuery?

Reply:
Currently `DAGDriver` is still using this function.:(